libasynql/src/poggit/libasynql/generic/SqliteStatementImpl.php

```diff
@@ -73,6 +73,9 @@ protected function formatVariable(GenericVariable $variable, $value, ?string $pl
 
 			case GenericVariable::TYPE_STRING:
 				assert(is_string($value));
+				if(strstr($value, "\0") !== false){
```

Comment:
Why use `strstr` instead of `strpos`?

Reply:
fixed in 2a2f82d0939660837ffcac2e2cebd4f50628fc37